.github/ISSUE_TEMPLATE/2-bot-commands.yml

```
@@ -10,6 +10,20 @@ body:
         
         A full list of bot commands is available in the [conda-forge webservices docs](https://conda-forge.org/docs/maintainer/infrastructure/#admin-web-services).
 
+        See below a quick summary of the available commands:
+
+          - @conda-forge-admin, please rerender: Reruns the feedstock rendering and pushes the changes to the PR.
+          - @conda-forge-admin, please add noarch: python: Adds `noarch: python` to the build configuration and rerenders the feedstock.
+          - @conda-forge-admin, please lint: Reruns the recipe linting check in the PR.
+          - @conda-forge-admin, please update team: Updates the feedstock maintenance team.
+          - @conda-forge-admin, please ping team: Mentions the feedstock maintenance team in the PR.
+          - @conda-forge-admin, please ping conda-forge: Mentions the relevant conda-forge team in the PR.
+          - @conda-forge-admin, please rerun bot: Adds the bot-rerun label to the PR, which reruns the migration and version update bot.
+          - @conda-forge-admin, please add bot automerge: Opens a PR to enable automerge for PRs from the auto-tick bot.
+          - @conda-forge-admin, please remove bot automerge: Opens a PR to disable automerge for bot PRs.
+          - @conda-forge-admin, please add user @username: Opens a PR to add a user to the feedstock maintainer team.
+          - @conda-forge-admin, please update version: Checks for new package versions and opens a PR with the necessary changes.
```

```suggestion
        See below a quick summary of the available commands for issue titles:

          - @conda-forge-admin, please rerender: Rerenders the feedstock in a new PR.
          - @conda-forge-admin, please add noarch: python: Adds `noarch: python` to the build configuration and rerenders the feedstock in a new PR.
          - @conda-forge-admin, please update team: Updates the feedstock maintenance team.
          - @conda-forge-admin, please add bot automerge: Opens a PR to enable automerge for PRs from the auto-tick bot.
          - @conda-forge-admin, please remove bot automerge: Opens a PR to disable automerge for bot PRs.
          - @conda-forge-admin, please add user @username: Opens a PR to add a user to the feedstock maintainer team.
          - @conda-forge-admin, please update version: Checks for new package versions and opens a PR with the necessary changes.
```

I realized that this list pertains just to commands that do things from issue titles. So I edited the list to only list the commands that work on issue titles. How does this look?

---

ah thanks. I went fast and didn't realize that. LGTM for me.